website/blog/2021-07-26-7.15.0.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "7.15.0 Released: Hack-style pipelines, TypeScript const enums and Rhino target support"
+author: Babel Team
+authorURL: https://twitter.com/babeljs
+date:   2021-07-26 0:00:00
+categories: announcements
+share_text: "Babel 7.15.0 Released"
+---
+
+This release enables parsing top-level await (Stage 4 at the [May](https://github.com/tc39/agendas/blob/master/2021/05.md) meeting) and transforming ergonomic brand checks for private fields (Stage 4 at the [July](https://github.com/tc39/agendas/blob/master/2021/07.md) meeting) by default. There is also now support for the [Hack-style](https://github.com/js-choi/proposal-hack-pipes) pipeline operator. We also improved our TypeScript support, implementing transform support for `const` enums and namespace aliases, and we expanded our heuristics to add `.displayName` to React components created by `React.createContext()` ([#13501](https://github.com/babel/babel/pull/13501)).
+
+We also introduced a new compiler assumption, [`noIncompleteNsImportDetection`](https://babeljs.io/docs/en/assumptions#noincompletensimportdetection), to produce a smaller output when compiling ECMAScript modules to CommonJS without worrying about partially initialized namespace imports caused by module cycles.
+
+Additionally, you can now specify [Rhino](https://mozilla.github.io/rhino/) as a compilation target.
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.15.0).
+
+<!-- truncate -->
+
+We're really grateful for all the support the community has shown over the last month's since our [funding post](https://babeljs.io/blog/2021/05/10/funding-update) update in May. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss about sponsorships!
```

Comment:
```suggestion
We're really grateful for all the support the community has shown over the last months, since our [funding post](https://babeljs.io/blog/2021/05/10/funding-update) update in May. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss about sponsorships!
```